akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -508,7 +513,9 @@ trait PhaseIsland[M] {
 
   def createPublisher(out: OutPort, logic: M): Publisher[Any]
 
-  def takePublisher(slot: Int, publisher: Publisher[Any]): Unit
+  def takePublisher(slot: Int, publisher: Publisher[Any], logic: M): Unit = takePublisher(slot, publisher)
+
+  protected def takePublisher(slot: Int, publisher: Publisher[Any]): Unit
```

Comment:
Yep, that's only so that I don't have to do wide-reaching changes just for trying things out.